Pipfile

```diff
@@ -5,7 +5,9 @@ verify_ssl = true
 
 [packages]
 # Do not add dependencies here, but list them in the setup.py instead
-ggshield = { editable = true, path = "." }
+ggshield = {file = "."}
+#TODO Replace with the newest version of py-gitguardian when it's released.
+pygitguardian = {editable = true, ref = "b567ad4faa707d7b9b99de713d2d615a2baf59e5", git = "git+https://github.com/GitGuardian/py-gitguardian.git"}
```

Comment:
These changes should not be necessary. What happens for you without them?

Reply:
The simplest solution for this is to do `pip install -e ../path/to/your/py-gitguardian`. Then you don't even need to commit changes.